indico/modules/files/util.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2024 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import os
+
+from indico.core.config import config
+
+
+def validate_upload_file_size(files, multiple=False):
+    """Validate size of one or more uploaded files.
+
+    Validation is done using the `MAX_UPLOAD_FILE_SIZE` and `MAX_UPLOAD_FILES_TOTAL_SIZE` config.
+    """
+    if not multiple:
+        files = [files]
+    max_upload_files_total_size = config.MAX_UPLOAD_FILES_TOTAL_SIZE * 1024 * 1024
```

Comment:
I just realized that this check is not useful: We already set the `MAX_CONTENT_LENGTH` based on this, so any request that would violate this fails much earlier (when flask realizes that the request is too big) with a "request entity too large" error...

Reply:
I removed the MAX_UPLOAD_FILES_TOTAL_SIZE check.